src/pages/FundraisingClaim/FundraisingClaim.tsx

```diff
@@ -3,45 +3,37 @@ import React, { useCallback, useEffect, useState } from "react"
 import { useAccount } from "wagmi"
 
 import { useSherClaimContract } from "../../hooks/useSherClaimContract"
+import { useGetFundraisePositionLazyQuery } from "../../graphql/types"
 
 export const FundraisingClaimPage = () => {
   const [{ data: accountData }] = useAccount()
   const sherClaim = useSherClaimContract()
-
   /**
-   * Amount of SHER tokens available to be claimed once the fundraise event ends.
+   * GraphQL query to fetch fundraise position
    */
-  const [claimableAmount, setClaimableAmount] = useState<BigNumber>()
+  const [getFundraisePotision, { data: fundraisePositionData, refetch: refetchFundraisePosition }] =
+    useGetFundraisePositionLazyQuery()
```

Comment:
Sweet, I didn't know hooks were automagically generated as well, for each query.  ❤️

Reply:
Yeah, this is really nice. And the result data is already typed.